ss/pebbledb/db.go

```diff
@@ -676,6 +690,8 @@ func (db *Database) ReverseIterator(storeKey string, version int64, start, end [
 	var upperBound []byte
 	if end != nil {
 		upperBound = MVCCEncode(prependStoreKey(storeKey, end), 0)
+	} else {
+		upperBound = MVCCEncode(prefixEnd(storePrefix(storeKey)), 0)
```

Comment:
should the version here be 0, or `version`?

Reply:
@codchen Should still be 0, we want the smallest possible key that strictly greater than every real key that starts with storePrefix(storeKey), only possible with passing in version 0 and using `prefixEnd`

Reply:
gotcha that makes sense